client_example.py

```diff
@@ -1,5 +1,5 @@
 from flask_ml.flask_ml_client import MLClient
-from flask_ml.flask_ml_server.constants import DataTypes
+from flask_ml.flask_ml_server.models import DataTypes

```

Comment:
`DataTypes` was included as part of the openapi spec, and so gets automatically generated